settings/Constants.ts

```diff
@@ -1,6 +1,7 @@
 export const RC_USER_ID      = 'rc_user_id';
 export const RC_ACCESS_TOKEN = 'rc_access_token';
 export const RC_CRM_URL      = 'rc_crm_url';
+export const RC_TIMEOUT      = 'rc_timeout';
```

Comment:
We confused this constant apparently, this shouldn't be PUSH_TIMEOUT? (look above at i18n file)

Reply:
Actually, this should be a default timeout value for every request, both rocketchat and rapidpro